### ISSUE TITLE
Implementation details. 

### ISSUE THREAD

**Issue by gangweiX** [guanyingc/DeepHDRVideo#24]
你好，在你的论文里，你说 “We first trained the CoarseNet with 10 epochs using a batch size of 16, and then trained the RefineNet with 15 epochs us- ing a batch size of 8. The learning rate was initially set to 0.0001 and halved every 5 epochs for both networks. We then end-to-end finetuned the whole network for 2 epochs using a learning rate of 0.00002.” 

但是，你的代码里默认每个阶段都训练30个epoch？请问复现您论文里的结果是按照论文里写的还是代码里默认的？非常期待您的回复。

**Comment by guanyingc** [comment]
Hi, please follow the setup described in the paper. Thanks.

**Comment by gangweiX** [comment]
Thanks.
